### PR TITLE
fix machine users UIDs across Galaxy VMs

### PR DESCRIPTION
**Pull request by jlqfab**
Make UIDs for machine users constant across Galaxy VMs. Loosely divided UIDs into a few ranges for easier management of UID numbers when adding and removing users: infrastructure team, support team, tools team, other.

### CODE REVIEW

**Comment by cat-bro** [comment]
Cool!  The machine_users dict is repeated in group_vars/dev.yml as a quick-fix for giving mike and tom sudo privileges on dev machines.  So to make this work on dev the IDs would need to be copied into that file too.

**Comment by jlqfab** [comment]
requested changes made

**Comment by jlqfab** [comment]
added code to remove old machine_user accounts - defined in group_vars/{all.yml,dev.yml}

File: roles/common/tasks/machine_users.yml [hunk 1]
```diff
@@ -3,17 +3,80 @@
...
+      - group
+
+  # FIX for deprecated code that added user files in /etc/sudoers.d/ - can be removed in future
+  - name: delete /etc/sudoers.d/username files for machine users
+    file:
+      path: "/etc/sudoers.d/{{ item.name }}"
+      state: absent
+    with_items:
+      - "{{ remove_machine_users }}"
+      - "{{ machine_users }}"
```
Comment (nuwang) [inline roles/common/tasks/machine_users.yml:22]: Concat lists?
```suggestion
    with_items: "{{ remove_machine_users }}" + "{{ machine_users }}"
```

**Review by cat-bro — APPROVED** [review]

**Pull request merged**